mlflow/metrics/genai/metric_definitions.py

```diff
@@ -68,6 +68,65 @@ def correctness(
     )
 
 
+def strict_correctness(
+    model: Optional[str] = None,
+    metric_version: Optional[str] = None,
+    examples: Optional[List[EvaluationExample]] = None,
+) -> EvaluationMetric:
+    """
+    This function will create a genai metric used to evaluate the strict correctness of an LLM
+    using the model provided. Strict correctness should be used in cases where correctness is
+    binary, and it will be assessed on whether the output aligns exactly with ground truth.
+
+    The ground_truth variable must be provided as part of the input dataset or output predictions.
+    This can be mapped to a column of a different name using the evaluator_config.
```

Comment:
can we add some pointers to help clarify when to use `strict_correctness` vs `correctness` ?